### ISSUE TITLE
Need way to track and deprovision virtual machines, not just redirectors

### ISSUE THREAD

**Issue by khyberspache** [preludeorg/operator-support#382]

**Comment by privateducky** [comment]
Can you give some more context on this?

The deprovisioning in Cloud/Connect plugins is centered around redirectors. Do you mean mean add deprovisioning to the Linux/Windows VMs you can spin up from the plugins or something else?

**Comment by privateducky** [comment]
This seems most solvable by auto-deleting compromised servers when the redirector itself is destroyed.

**Comment by khyberspache** [comment]
@privateducky I'm finding the "real" use case for those linux servers & windows servers is as external staging/resource/initial access systems. 

It would be great to track them the way we do redirectors so I can see how many attack boxes I would have in my fleet.

**Comment by privateducky** [comment]
gotcha; would FQDN of each be sufficient (with the ability to terminate them)? trying to find the appropriate line of functionality w/out becoming an AWS console :)

**Comment by khyberspache** [comment]
Yeah 100%, literally a list, with platform, and a delete button. Not functionality just a "here's what you have"

**Comment by privateducky** [comment]
This will be handled (most likely) in 1.6, as we'll be focused on AWS/cloud integration points.
